[ADLS] Adding note about service principal group membership

matt1883 opened the pull request:
[ADLS] Adding a note about how to add service principals to groups in Azure AD.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

matt1883 committed:
[ADLS] Adding note about service principal group membership

[ADLS] Adding a note about how to add service principals to groups in Azure AD.

matt1883 committed:
Update data-lake-store-secure-data.md

Adding link to Azure AD PS module doc
